### ISSUE TITLE
[Accessibility][PDF Viewer] Content is lost at 320px width 

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1750]
**Steps to Reproduce**
1. Open page in Google Chrome.
2. With Chrome developer tools open and docked to the side, resize page to 320px width.

**Current Behavior**
At 320 pixels width, pages require horizontal scrolling in addition to vertical scrolling, and the search controls  and/or the control dock content is cut off. In the In-Line example, the control dock is cut off on both sides. In the Sized Container example, the search input is too narrow to view the user's input. In the Full-Width example, the Adobe Acrobat icon overlaps the current page input.

Users who require high browser zoom levels to view content may be unable to access some functionality.

**Expected behavior/code**
Ensure that smaller screens are supported by reflowing content as necessary to allow for an equivalent experience to larger screens without requiring two dimensional scrolling.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Users with low vision, users with limited mobility

**WCAG SC**
1.4.10 Reflow

**Screenshot**
![In the In-Line example, at 320px width, control dock is cut off on both sides.](https://user-images.githubusercontent.com/86849351/128521650-74e9e438-9e8e-4482-a4f3-8337bc9227f6.png)
![In the Sized Container example, at 320px width, search input is too narrow to view user's input](https://user-images.githubusercontent.com/86849351/128521653-e1843932-bd94-486e-88ea-ee414413eb1f.png)
![In the Full-Width example, at 320px width, Adobe Acrobat icon overlaps the current page input.](https://user-images.githubusercontent.com/86849351/128521654-0eeb8b3b-4593-4d1c-a402-c4cf114e8326.png)

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by glasert** [comment]
This issue won't be fixed as the acrobat team is working on a modern viewer which addresses accessibility and will replace the current viewer in 2022. Therefore we defer and close this issue
